doc/terminal-v1-timeline.md

```diff
@@ -0,0 +1,91 @@
+* Issues/features/asks not on our list of v1.0 features is assigned to the [Terminal Backlog](https://github.com/microsoft/terminal/milestone/7) for subsequent triage, prioritization & scheduling.
+
+## v1.0 Features
+
+The following are a list of the key features/feature-areas we're aiming to deliver for Terminal v1.0.
+
+> 👉 Note: There are many other features that don't fit within v1.0, but will be re-assessed and prioritized for v2.0, the plan for which will be published in early in 2020.
+
+| Release | Priority\* | Feature | Description/Notes |
+| --- | --- | --- | --- |
+| V1 | 0 | Performance & Efficiency | Terminal shall be fast and efficient. Input latency should be eliminated wherever possible. Terminal will be very memory-efficient, and will avoid utilizing unnecessary dependencies to minimize memory consumption and disk footprint |
+| V1 | 0 | Reliability | Every reasonable step should be taken to ensure that Terminal will not crash unexpectedly. Crashing is considered harmful to the user's well-being & state of mind. Crashing issues are prioritized Pri-0 by default |
+| V1 | 0 | Code Reuse | Terminal's core engine will re-use & share componentry from within Windows Console wherever feasible to minimize support & maintenance costs for both|
+| V1 | 0 | Terminal Reuse | Terminal's core will be hostable as a UWP (and perhaps WPF) Control so that apps can host/embed a high quality Terminal. This will satisfy a long-standing ask from many customers and partners for a hostable/embeddable Terminal Control. |
+| V1 | 0 | Rich, modern text renderer | Terminal must be able to render glyphs from East Asian and Middle Asian languages, inc. Chinese, Hebrew, Arabic, etc. Terminal will also be able to render Emoji - an increasingly important feature consdiering that several programming languages now support Emoji in method and variable names! To render such glyphs, the Terminal needs a DirectWrite-based layout & rendering system which supports font-fallback, customizable text-layout, GPU accelerated rendering, and many other features not currently supported by the built-in Windows Console |
+| V1 | 0 | Unicode & UTF-8 support | Terminal must be able to store data encoded as Unicode UTF-16/UCS-2 and UTF-8, including surrogate pairs |
+| V1 | 0 | Extended Unicode Support | The Terminal will support rendering text for almost every language for which there is a fixed-width font including East Asian languages. Bonus points for RTL languages/scripts. |
+| V1 | 0 | Multiple-instances | Must be able to launch multiple independent instances of the Terminal - users need to run tools side-by-side independently |
+| V1 | 0 | Elevation | Terminal can be launched "elevated" with Admin rights if required so that the user can perform operatoins that affect machine-wide state |

```

Comment:
```suggestion
| V1 | 0 | Elevation | Terminal can be launched "elevated" with Admin rights if required so that the user can perform operations that affect machine-wide state |
```